verl/trainer/ppo/ray_trainer.py

```diff
@@ -1380,7 +1387,18 @@ def fit(self):
                             self._save_checkpoint()
 
                 with marked_timer("stop_profile", timing_raw):
-                    self._stop_profiling(do_profile)
```

Comment:
could you move local var to `self._xxx` and L1390-L1401 to `self._stop_profiling`, and similarly for `self._start_profiling`?